apis/medusa/v1alpha1/cassandrabackup_types.go

```diff
@@ -18,20 +18,13 @@ package v1alpha1
 
```

Comment:
If we intend to replace this api with medusabackup_types, then I think we need to discuss how we want to handle breaking API changes. That is _not_ a discussion that I want to have in the PR review 😄 Do you think though that some comments should be added to indicate that the API is deprecated?

Reply:
very much so. I had looked in the kubebuilder book if there was any annotation we could put to mark stuff as deprecated, with no luck.
But I just came across this: https://github.com/kubernetes-sigs/kubebuilder/issues/2116

Turns out the annotation exists but isn't documented: //+kubebuilder:deprecatedversion:warning="example.com/v1alpha1 CronTab is deprecated"`

I'll use this.

Reply:
perfect 🙂